Makefile

```
@@ -37,8 +37,8 @@ test-cloud: ## Run only the cloud-based tests (requires cloud Grafana instance a
 
 .PHONY: run
 run: ## Run the MCP server in stdio mode.
-	go run ./...
+	go run ./cmd/mcp-grafana/...
```

```suggestion
	go run ./cmd/mcp-grafana
```

You don't need the ... anymore since the command is specifically pointing to the package of interest.